tests/features/site-branding.feature

```diff
@@ -72,3 +72,41 @@ Feature: Site branding
     Then I should see the "page header" element in the "page"
     When I am on "the user registration page"
     Then I should see the "page header" element in the "page"
+
+  @javascript @enable-non-eu-language
+  Scenario Outline: The European Union header logo has accessibility attributes.
+    Given I am on the homepage
+    And the theme is configured to use the "European Union" style
+    When I open the language switcher dialog
+    And I click "<language_selector>"
+    Then the header logo should contain accessibility attributes with link "<link>" label "<label>" alt "<alt>" title "<title>"
+    When I am on "the user registration page"
+    And I open the language switcher dialog
+    And I click "<language_selector>"
+    Then the header logo should contain accessibility attributes with link "<link>" label "<label>" alt "<alt>" title "<title>"
+
+    Examples:
+      | language_selector | link                                      | label                 | alt                 | title          |
+      | English           | https://european-union.europa.eu/index_en | Home - European Union | European Union flag | European Union |
+      | български         | https://european-union.europa.eu/index_bg | Home - European Union | European Union flag | European Union |
+      # Non-EU language.
+      | Icelandic         | https://european-union.europa.eu          | Home - European Union | European Union flag | European Union |
```

Comment:
Label, alt and title are exactly the same for each language so why can't we just adapt the existing scenarios testing the logos for every language already? Now we have two different scenarios for each component library testing the exact same thing the only difference being the URL.